frontend/src/components/announcements/CreateAnnouncementBlock.js

```diff
@@ -63,6 +66,7 @@ function CreateAnnouncementBlock(props) {
     return(
         <div>
             <h1 className={styles.AnnouncementTitle} > Create Accouncements</h1>
+            <Link to ="/announcements" className={styles.createButton}>Return to Announcements</Link>
```

Comment:
In terms of UX, usually a back arrow is used instead. I am sure material-ui has an icon for it that you can use with an IconButton. Search for Icon button and it should come up in some calendar component as an example.